tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1563,4 +1563,46 @@ public function userGetsAllUsersOfTwoGroupsUsingTheGraphApi(string $user, string
 		);
 		$this->featureContext->setResponse($response);
 	}
+
+	/**
+	 * @When /^the administrator "([^"]*)" adds the following users to a group "([^"]*)" in a single request using the Graph API$/
+	 *
+	 * @param string $user
+	 * @param string $group
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 * @throws GuzzleException
+	 */
+	public function theAdministratorAddsTheFollowingUsersToAGroupInASingleRequestUsingTheGraphApi(string $user, string $group, TableNode $table): void {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		$this->featureContext->verifyTableNodeColumns($table, ['username']);
+		$usersId = [];
+
+		try {
+			$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
+		} catch (Exception $e) {
+			$groupId = WebDavHelper::generateUUIDv4();
+		}
+
+		foreach ($table->getHash() as $row) {
+			try {
+				$usersId[] = $this->featureContext->getAttributeOfCreatedUser($row['username'], "id");
+			} catch (Exception $e) {
+				$usersId[] = WebDavHelper::generateUUIDv4();
+			}
+		}
```

Comment:
Let the step only do the positive cases. If we need to test trying to add users to non-existing group then we can create another different step.
```suggestion
		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");

		foreach ($table->getHash() as $row) {
			$usersId[] = $this->featureContext->getAttributeOfCreatedUser($row['username'], "id");
		}
```

Reply:
I was going to add a scenario where we need this code